login/login.css

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  overflow: hidden;
+}
+
+.main {
+  min-height: 100%;
+  margin: 0 auto;
+  width: 100%;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;

```

Comment:
you dont have to always use all of them , aling-items , justify-content , aling-content : center if you don't need it